contracts/smartwallet/CustomSmartWallet.sol

```diff
@@ -57,7 +57,7 @@ contract CustomSmartWallet is IForwarder {
         }
     }
 
-    function getOwner() private view returns (bytes32 owner){
+    function getOwner() public view override returns (bytes32 owner){
```

Comment:
This change is already included in the ethers branch, so I just include it, here.

Reply:
not public, but we need to change its accessibility so I decided to do the same change that we applied before.